File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/FeatureMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.Feature;
import pl.yalgrin.playnite.simplesync.library.dto.FeatureDTO;

@Component
public class FeatureMapper extends AbstractObjectMapper<Feature, FeatureDTO> {

    @Override
    protected FeatureDTO createDTO() {
        return new FeatureDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/FeatureMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.Feature
import pl.yalgrin.playnite.simplesync.library.dto.FeatureDTO

@Component
class FeatureMapper : LibraryObjectMapperImpl<Feature, FeatureDTO>() {
    override fun createDTO(): FeatureDTO = FeatureDTO()
}